src/libs/actions/User.js

```diff
@@ -70,6 +71,8 @@ function closeAccount(message) {
             ],
         },
     );
+    // Removing callbacks/listener which are added after login. Fixes the issue user's not able to login after app came to active from inactive
```

Comment:
```suggestion
    // Run cleanup actions to prevent reconnection callbacks from blocking logging in again
```

Maybe something like this would be a bit more concise.